test/check-machines-settings

```diff
@@ -390,24 +390,24 @@ class TestMachinesSettings(VirtualMachinesCase):
         b.click("#cpu-config-dialog-apply")
         b.wait_visible("#cpu-tooltip")
 
+        dom_xml = "virsh -c qemu:///system dumpxml --inactive subVmTest1"
+
+        xmllint_elem = "{0} | xmllint --xpath 'string(//domain/cpu/@mode)' - 2>&1 || true".format(dom_xml)
+        wait(lambda: "host-passthrough" in m.execute(xmllint_elem).strip())
```

Comment:
Was moving that check from "after shutting off" to "still running" the magic trick to pacify the test? :grin: This feels unrelated to virt-xml, unless I'm missing something? Not a biggie, but might be cleaner to split it into a separate commit.

Reply:
Actually not - I can just revert this.

Reply:
This was not needed, reverted.